src/goto-programs/interpreter.cpp

```diff
@@ -862,7 +862,7 @@ void interpretert::build_memory_map()
 {
   // put in a dummy for NULL
   memory.resize(1);
-  inverse_memory_map[0]="NULL-OBJECT";
+  inverse_memory_map[0]=ID_null_object;
```

Comment:
I think this change is fine. Strings used exactly once in the code base are very suspicious and should likely all be fixed.